infrastructure/builder/templates/k8s/secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: ${projectName}
+data:
+  $merge:
+    $map: {$eval: secrets}
+    each(s):
+      ${uppercase(s.key)}: '{{ ${s.val} | toJson | trimAll "\"" | b64enc }}'
```

Comment:
I don't think the `trimAll` will be compatible with `!env:json` config..

Reply:
It only trims leading/trailing `"`. It seems to work empirically so far. This whole line is very ugly though and I won't be surprised when it doesn't work in some other way.